docs/algorithms/apply_gate_library.rst

```diff
@@ -10,3 +10,4 @@ in the same :ref:`technology <fcn-cell-technologies>` as the provided gate libra
 implementations for each gate present in the passed ``gate_level_layout``.
 
 .. doxygenfunction:: fiction::apply_gate_library(const GateLyt& lyt)
+.. doxygenfunction:: fiction::apply_dynamic_gate_library
```

Comment:
@marcelwa Something is broken here, but I do not know why. I have already tried several things. Maybe you have an idea 🙏🏼

Reply:
The function has a very funky signature. Maybe Doxygen cannot infer it automatically. Have you tried specifying it here completely?